### ISSUE TITLE
[6.4.0-rc.1] console warning in @storybook/testing-react tests

### ISSUE THREAD

**Issue by IanVS** [storybookjs/testing-library#3]
**Describe the bug**
I have a story file with two stories, one is a CSF2 component which is imported into a unit test file using `@storybook/testing-react`'s `composeStories()` method, and the other is a CSF3 component with a play function.  

In my unit test file, I import `screen` from `@testing-library/react`, and use it in my unit test.

I noticed now that when I import anything from `@storybook/testing-library` in my story file for use in my play function, I get this warning in the console:

```
You are using Testing Library's `screen` object. Use `within(canvasElement)` instead.
More info: https://storybook.js.org/docs/react/essentials/interactions
```

The warning occurs regardless if the imported method in the story file is actually used in the story file, and it doesn't seem to matter what method is imported either.  

**To Reproduce**
https://github.com/IanVS/storybook-issue-16677

**Comment by ghengeveld** [comment]
Looks like we're mutating the original `screen` object of `@testing-library/react` rather than copying it in the instrumenter. The problem is not with us [overriding](https://github.com/storybookjs/testing-library/blob/e8435f609a94beaddacb240b86caf3cc893d61e6/src/index.ts#L31-L43) the `screen` object in our wrapper, but rather with `instrument` returning the original `screen` object in the first place.

**Comment by yannbf** [comment]
Hey @IanVS this issue should be fixed in v0.0.4! 🚀

**Comment by IanVS** [comment]
Yep, seems all set, thanks!
